### PR TITLE
Creating data model

### PR DESCRIPTION
**Pull request by BANCUNGUYE66**
### Blog app

- [x] Give my app a meaningful name.
- [x] Project has Postgres database set up.
- [x] Build project schema.
- [x] Create and run the necessary migration files.
- [x] Table and column names match the ERD diagram.
- [x] All columns that are foreign keys have a corresponding index.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,133 @@
...
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank microverse community for the support and guidance.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
Comment (emmanuellekamwa) [inline README.md:132]: - [ ] Your MIT license link is broken. Please kindly fix that.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ⛔❌**
Hello @BANCUNGUYE66 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Descriptive PR.
- Good commit messages.
- Correct git-flow.

## Required Changes ♻️
_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @BANCUNGUYE66, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made Requested Changes By Previous Reviewer :heavy_check_mark: 
: 

## Optional suggestions
- [ ] It would be good if you set up your foreign key for the `post`, `like` and `comment` tables with a pointer to which table it is hooked to;
e.g
`:author, foreign_key: { to_table: 'users' }`

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by danmainah** [comment]
Hi @BANCUNGUYE66  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️  Added all tables required for this milestone

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**